packages/router/src/lib/content.ts

```diff
@@ -0,0 +1,17 @@
+import { map } from 'rxjs';
+import { injectActivatedRoute } from './define-route';
+
+const posts = import.meta.glob(['/src/content/**/*.md'], {
+  as: 'raw',
+  eager: true,
+});
+
+export function injectContent(param = 'slug') {
+  const route = injectActivatedRoute();
+  const content = route.paramMap.pipe(
+    map((params) => params.get(param)),
+    map((slug) => posts[`/src/content/${slug}.md`] || 'No Content Found')
```

Comment:
It would be nice if we could inject our own "text" instead of `No Content Found` (keep the default behavior if the custom "text" is not injected)

Reply:
I was thinking of creating a service alongside this component to handle rendering the markdown. This message could go alongside it

Reply:
Good point